src/plugins/telemetry/public/plugin.ts

```diff
@@ -155,10 +148,12 @@ export class TelemetryPlugin implements Plugin<TelemetryPluginSetup, TelemetryPl
       telemetryConstants = getTelemetryConstants(docLinks);
     });
 
-    analytics.registerShipper(ElasticV3BrowserShipper, {
-      channelName: 'kibana-browser',
-      version: currentKibanaVersion,
-      sendTo: config.sendUsageTo === 'prod' ? 'production' : 'staging',
+    import('@kbn/analytics-shippers-elastic-v3-browser').then(({ ElasticV3BrowserShipper }) => {
+      analytics.registerShipper(ElasticV3BrowserShipper, {
+        channelName: 'kibana-browser',
+        version: currentKibanaVersion,
+        sendTo: config.sendUsageTo === 'prod' ? 'production' : 'staging',
+      });
     });
```

Comment:
Given this block will be unconditionally executed, may I ask what the lazy import bring here apart from generating additional requests and artificially reducing the initial load bundle metric?